shuffle/cli/src/shared.rs

```diff
@@ -1031,4 +1053,105 @@ mod test {
             .unwrap();
         assert_eq!(home.check_networks_toml_exists().is_err(), false);
     }
+
+    fn post_txn_json_output() -> Value {
```

Comment:
nit: As you've mentioned, `shared.rs` is getting big. Let's pull out everything related to `DevApiClient` into a new file `dev_api_client.rs`,including `impl`, `struct` instance methods, and all tests and test helpers. This would fit well into its own commit on top of what's already done here.

Reply:
Great idea! Will do